### PR TITLE
Resolving timing issue test

### PR DESCRIPTION
**Pull request by zalokhan**
Giving more time and multiple to check if this was a timing issue and the failure persists.
(Need to check multiple times on travis env so please do not merge)

### CODE REVIEW

**Comment by zalokhan** [comment]
I've run this quite many times now and it doesn't fail on travis. So I feel its safe to say that the test was failing due to a timeout issue. This patch should fix it.
Resolving #287 

Interestingly, the other test handleFailures in ManagementClientTest continues to fail transiently.
I've narrowed down to the cause of the hanging of this test case:
This test has 2 statements -> bootstrap management server and then send failure msg.
So to recreate the scenario,
The bootstrap handler in Management Server needs to have a small sleep after sending the ack as follows:
  ```
  @ServerHandler(type = CorfuMsgType.MANAGEMENT_BOOTSTRAP)
    public synchronized void handleManagementBootstrap(CorfuPayloadMsg<Layout> msg, ChannelHandlerContext ctx, IServerRouter r) {
        log.info("Received Bootstrap Layout : {}", msg.getPayload());
        safeUpdateLayout(msg.getPayload());
        r.sendResponse(ctx, msg, new CorfuMsg(CorfuMsgType.ACK));
        try {
            Thread.sleep(3000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
```
The ack is received by the test case and it then sends the failure message which is somehow lost/blocked.
The server then gets busy in trying to connect to a layout server which doesn't exist at test:9000.

File: test/src/test/java/org/corfudb/infrastructure/PeriodicPollPolicyTest.java [hunk 1]
```diff
@@ -98,13 +109,18 @@ public void failedPolling() throws InterruptedException {
         // A little more than responseTimeout for periodicPolling
         Thread.sleep(1100);
```
Comment (no2chem) [inline test/src/test/java/org/corfudb/infrastructure/PeriodicPollPolicyTest.java]: Is there a specific reason we need this timeout?

Comment (zalokhan) [inline test/src/test/java/org/corfudb/infrastructure/PeriodicPollPolicyTest.java]: No we can get rid of this now.
I'll fix this.

**Comment by slfritchie** [comment]
FWIW, I see zero failures in 20 runs, max runtime of just over 6 seconds (PeriodicPollPolicyTest) and 5 msec (ManagementClientTest) with little jitter.

**Comment by zalokhan** [comment]
The bug in ManagemenClientTest is not yet fixed. I've just identified the issue.
I'll push in a separate patch for that may be.

**Comment by no2chem** [comment]
Ok, mark it as ready if you're ready to merge it.

**Pull request merged**